simplemonitor/Alerters/mail.py

```diff
@@ -62,7 +61,7 @@ def send_alert(self, name: str, monitor: Monitor) -> None:
             AlertLength.NOTIFICATION, alert_type, monitor
         )
         body = self.build_message(AlertLength.FULL, alert_type, monitor)
-        message.attach(MIMEText(body, "plain"))
+        message.set_content(body)
```

Comment:
I don't know if there are cases where we're sending anything other than plain ASCII text? but the tests pass 🤷